docs/docker.md

```diff
@@ -62,14 +62,8 @@ More details about node configuration can be found in [configuration documentati
 
 ### Persisting Data
 
-To persist blockchain data between container runs use [Docker volumes](https://docs.docker.com/engine/admin/volumes/volumes/) and enable chain persistence in your configuration:
-
-```yaml
-# ... SNIP
-chain:
-    persist: true
-# SNIP ...
-```
+The blockchain data is persisted by default, inside the Docker container.
```

Comment:
It's worth nothing mentioning that by default the database is persisted at: `/home/aeternity/node/data/mnesia` in the container.

Reply:
I expect it benefits Docker image user re which parameters to tune when starting container. User may be surprised to find that the new container fails after a while if container started with large memory but little storage.

Reply:
(I had not got initial comment, you suggested change now included. 👍 )